_data/simple-icons.json

```
@@ -310,6 +310,11 @@
             "hex": "18BFFF",
             "source": "https://airtable.com/press"
         },
+        {
+            "title": "Alacritty",
+            "hex": "F46D01",
+            "source": "https://github.com/alacritty/alacritty/tree/master/extra/logo"
```

I'd suggest we're explicit with which version of the logo is used, including the full path (including version).
```suggestion
            "source": "https://github.com/alacritty/alacritty/blob/6d8db6b9dfadd6164c4be7a053f25db8ef6b7998/extra/logo/alacritty-simple.svg"
```